eva_cttv_pipeline/clinvar_record.py

```diff
@@ -86,20 +92,27 @@ def ensembl_id(self):
                         i = 0
                         while i < len(symbol):
                             try:
-                                return ClinvarRecord.cached_symbol_2_ensembl[symbol[i]['elementValue']['value']]
+                                return ClinvarRecord.cached_symbol_2_ensembl[
+                                    symbol[i]['elementValue']['value']
+                                ]
                             except KeyError:
                                 not_solved = True
                                 while not_solved:
                                     try:
-                                        url = 'http://rest.ensembl.org/lookup/symbol/homo_sapiens/' + symbol[i]['elementValue']['value'] + '?content-type=application/json'
-                                        rawreply = urllib.request.urlopen(url).read()
-                                        ensembl_json = json.loads(rawreply.decode())
+                                        url = 'http://rest.ensembl.org/' + \
+                                              'lookup/symbol/homo_sapiens/' + \
+                                              symbol[i]['elementValue']['value'] + \
+                                              '?content-type=application/json'
+                                        raw_reply = urllib.request.urlopen(url).read()
+                                        ensembl_json = json.loads(raw_reply.decode())
```

Comment:
I think this is from when the file was in a different structure, but just hadn't been removed.